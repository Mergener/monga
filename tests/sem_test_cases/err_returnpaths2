function foo(): float {
    while (1 >= 1) {
        return 4.0;
    }

    if (2 < 3) {
        return 6.0;
    }
}