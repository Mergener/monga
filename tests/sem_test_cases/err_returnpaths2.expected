[Error] Module tests/sem_test_cases/err_returnpaths2, line 9, column 2: Not all paths are returning a value.
Semantic errors were caught during analysis.
